fppanalysis/two_dim_velocity_estimates.py

```diff
@@ -113,6 +209,15 @@ def estimate_velocities_for_pixel(x, y, ds: xr.Dataset):
     of the maximums of the two cross-correlations involved (good luck
     understanding this last sentence :))
```

Comment:
Modify the docstring so that the method description is documentet in an if method == 'cross_corr', as you did for cond_av